.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
       - id: isort
         name: isort
         language: system
-        entry: isort -y
+        entry: isort
```

Comment:
Unrelated change - can you split it into a separate commit?

Reply:
i did.